flu/flu_solution_1.R

```diff
@@ -0,0 +1,28 @@
+# initial conditions
```

Comment:
elsewhere we don't have an underscore as `_1`, `_2`, etc.  These are also in the wrong directory - needs to be in `files/flu` not `flu`

Reply:
Updated - as with other flu renaming/moving